tests/modelling/base.py

```diff
@@ -48,15 +48,16 @@ class BaseFFCTestCase(TestCase):
     def setUp(self):
         self.__calendar = date_range('2014', '2015', freq=trading_day)
         self.__assets = assets = Int64Index(arange(1, 20))
-        self.__finder = AssetFinder(
-            make_simple_asset_info(
+
+        # Set up env for test
+        self.__env = TradingEnvironment()
+        self.__env.write_data(
+            equities_df=make_simple_asset_info(
                 assets,
                 self.__calendar[0],
                 self.__calendar[-1],
-            ),
-            db_path=':memory:',
-            create_table=True,
-        )
+            ))
+        self.__finder = self.__env.asset_finder
```

Comment:
We shouldn't be storing `__env` on self if nothing else uses it.

Also :-1: on the fact that I now need to construct an entire `TradingEnvironment` in order to make an `AssetFinder`.